src/Tests/Tests/XPack/Slm/SlmApiTests.cs

```diff
@@ -102,6 +103,18 @@ public SlmApiTests(XPackCluster cluster, EndpointUsage usage) : base(new Coordin
 						(v, c, r) => c.SnapshotLifecycleManagement.ExecuteSnapshotLifecycleAsync(r)
 					)
 			},
+			{
+				ExecuteRetentionLifecycleStep, u =>
+					u.Calls<ExecuteRetentionSnapshotLifecycleDescriptor, ExecuteRetentionSnapshotLifecycleRequest, IExecuteRetentionSnapshotLifecycleRequest,
+						ExecuteRetentionSnapshotLifecycleResponse>(
+						v => new ExecuteRetentionSnapshotLifecycleRequest(),
+						(v, d) => d,
+						(v, c, f) => c.SnapshotLifecycleManagement.ExecuteRetentionSnapshotLifecycle(f),
```

Comment:
Should this just be `.SnapshotLifecycleManagement.ExcuteRetention()` based on the API name

Reply:
Good call, done.